userprofile/factories.py

```diff
@@ -36,7 +37,7 @@ class Meta:
         model = User
         django_get_or_create = ('username',)
 
-    username = "edx"
+    username = factory.Sequence(lambda n: 'user_%d' % n)
```

Comment:
@ahmedaljazzar instead of doing this, can we just use the github_username parameter on make_user_and_organization function be the username?

Reply:
Yes we can do that, but don't you think this might confuse stuff? I think this is okay as we can create as many object as we want, and we can override its value whenever we want.

Reply:
@ahmedaljazzar confuse stuff in which way? I'm not sure I follow.

I think it's more readable if it's a parameter, you can re-use the github_username parameter or if you think that's confusing it can be another parameter "username".